nntrainer/layers/cl_layers/rmsnorm_layer_cl.cpp

```diff
@@ -339,7 +330,9 @@ void RMSNormLayerCl::incremental_forwarding(nntrainer::RunLayerContext &context,
   if (in_step.getDataType() == ml::train::TensorDim::DataType::FP32) {
     rmsnormProcess(in, out, gamma, epsilon);
   } else {
+#ifdef ENABLE_FP16
     rmsnormProcess_fp16(in, out, gamma, epsilon);
+#endif
```

Comment:
it might be good to throw an error when fp16 is not enabled!

Reply:
Thank you for the comment. I updated it :)